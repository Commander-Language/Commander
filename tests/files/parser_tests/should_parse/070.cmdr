//Tests array stuff gets parsed correctly
arr = [1, 2, 3];
length = arr.length();
second = arr[1];
