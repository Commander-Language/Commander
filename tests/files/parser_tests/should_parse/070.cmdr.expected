(PrgmNode
	(ExprStmtNode (BinOpExprNode (BindingNode arr) SET (ArrayExprNode (ExprsNode (IntExprNode 1) (IntExprNode 2) (IntExprNode 3)))))
	(ExprStmtNode (BinOpExprNode (BindingNode length) SET (ApiCallExprNode (LValueExprNode (VarExprNode arr)) length (ExprsNode))))
	(ExprStmtNode (BinOpExprNode (BindingNode second) SET (LValueExprNode (IndexExprNode (VarExprNode arr) (IntExprNode 1))))))
